app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -601,7 +595,7 @@
 
 
     <!-- Sample text for accessibility font size -->
-    <string name="accessibility_text_size_sample_text">The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. 
+    <string name="accessibility_text_size_sample_text">The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog.
```

Comment:
Prefer: "The five boxing wizards jump quickly." ;o)

Reply:
I think this is outside the scope of the PR 😃 
Only changed this by accident since my editor strips out trailing spaces.